### ISSUE TITLE
Quotationmarks in attributes and option of fix functions

### ISSUE THREAD

**Issue by TobiasNx** [metafacture/metafacture-fix#74]
Concerning the attributes and options stated in lookuo this brings me to a broader topic: Is there a rule when to use double/single quotation marks or not. The path is not using any, the values-pair in the lookup list does. Is there a rule/concvention in catmandu how to do this.

```
lookup(n,
  'Goethe': '118540238',
  'Schiller': '118607626',
  '__default': '4428933-9')
```

_Originally posted by @TobiasNx in https://github.com/metafacture/metafacture-fix/issues/50#issuecomment-968637211_

Other examples from catmandu:

`replace_all(authors.*,"a","AB")`

`select any_match(years,2005)`

`lookup("title","dict.csv", sep_char:'|',default:'NONE')`

`sort_field(tags, reverse:1)`

**Comment by blackwinter** [comment]
Let me piggy-back on this with raising my disappointment that `var` (e.g., in `do list(path: '...', var: '...')`) **must** be quoted since, apparently, it's a [reserved keyword](https://www.eclipse.org/Xtext/documentation/305_xbase.html#xbase-syntax-keywords) in our grammar :disappointed: 

As far as convention, I've almost resigned myself to just quoting everything :shrug:

**Comment by blackwinter** [comment]
Don't know about Catmandu, but my suggestion would be: Quote everything (with double quotes), except for option names (because it looks ugly), except for `var` (because we _have_ to quote that one).

**Comment by fsteeg** [comment]
@TobiasNx will add a section on `"` and `'` usage as part of #73.

Functional review: @blackwinter & @fsteeg

**Comment by TobiasNx** [comment]
Resolved by #141
